planetiler-core/src/main/java/com/onthegomap/planetiler/util/VarInt.java

```diff
@@ -0,0 +1,129 @@
+package com.onthegomap.planetiler.util;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+
+/**
+ * Protocol Buffers compatible VarInt encoding adapted from
+ * https://raw.githubusercontent.com/bazelbuild/bazel/master/src/main/java/com/google/devtools/build/lib/util/VarInt.java
+ *
+ * Copyright 2014 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
```

Comment:
For this adapted code, I've been putting the license at the top of the file, with the javadoc for the class used to give a description of what was changed (if anything).  See VectorTile.java as an example.

Reply:
Moved to the top; though at this point the code as-is _might_ be trivial enough to not require this...

Reply:
Thanks! And can you add a short javadoc comment with link to the original and say what changed (like un-unrolling the loop) from the original?